complex objects/cargo/src/lib.rs

```diff
@@ -55,13 +55,13 @@ pub unsafe extern "C" fn toodle_destroy(toodle: *mut Toodle) {
 }
 
 #[no_mangle]
-pub unsafe extern "C" fn toodle_logins(toodle: *mut Toodle) -> *mut Arc<LoginManager> {
+pub unsafe extern "C" fn toodle_logins(toodle: *mut Arc<Toodle>) -> *mut Arc<LoginManager> {
```

Comment:
How does refcounting work across the FFI boundary? My understanding was that it doesn't do anything special, so you're reliant on that `Arc` being held somewhere else, no?

Reply:
Yeah, I think I was just Arc happy at that point.